docs/extensions/guides/main-extension.md

```diff
@@ -20,11 +22,18 @@ export default class ExampleExtensionMain extends LensMainExtension {
 }
 ```
 
-There are two methods that you can implement to facilitate running your custom code. `onActivate()` is called when your extension has been successfully enabled. By implementing `onActivate()` you can initiate your custom code. `onDeactivate()` is called when the extension is disabled (typically from the [Lens Extensions Page]()) and when implemented gives you a chance to clean up after your extension, if necessary. The example above simply logs messages when the extension is enabled and disabled. Note that to see standard output from the main process there must be a console connected to it. This is typically achieved by starting Lens from the command prompt.
+Two methods enable you to run custom code: `onActivate()` and `onDeactivate()`. Enabling your extension calls `onActivate()` and disabling your extension calls `onDeactivate()`. You can initiate custom code by implementing `onActivate()`. Implementing `onDeactivate()` gives you the opportunity to clean up after your extension.
 
-The following example is a little more interesting in that it accesses some Lens state data and periodically logs the name of the currently active cluster in Lens.
+Disable extensions from the Lens Extensions page:
 
-``` typescript
+1. Navigate to **File** > **Extension** in the top menu bar.
```

Comment:
**Extensions**

Plus of course mac has to be different, so it's **Lens** > **Extensions**

Reply:
Thanks, Jim. I corrected this.